test/export/StructureDefinitionExporter.test.ts

```diff
@@ -418,6 +424,47 @@ describe('StructureDefinitionExporter', () => {
     expect(constrainedValue.type).toHaveLength(11);
   });
 
+  // FixValue rule
+  it('should apply a correct FixValueRule', () => {
+    const profile = new Profile('Foo');
+    profile.parent = 'Observation';
+
+    const rule = new FixedValueRule('code');
+    const fixedFshCode = new FshCode('foo', 'http://foo.com');
+    rule.fixedValue = fixedFshCode;
+    profile.rules.push(rule);
+
+    const sd = exporter.exportStructDef(profile, input);
+    const baseStructDef = sd.getBaseStructureDefinition();
+
+    const baseCode = baseStructDef.findElement('Observation.code');
+    const fixedCode = sd.findElement('Observation.code');
+
+    expect(baseCode.patternCodeableConcept).toBeUndefined();
+    expect(fixedCode.patternCodeableConcept).toEqual({
+      coding: [{ code: 'foo', system: 'http://foo.com' }]
+    });
+  });
+
+  it('should not apply an incorrect FixValueRule', () => {
+    // TODO: this should check for emitting an error once logging is set up
```

Comment:
I might be misunderstanding, but isn't logging already set up? Why can't we perform this test already?

Reply:
We have the logging set up, but we haven't set up the ability to check what errors are emitted directly in a test yet. That was going to be an additional feature to be added after the initial logging task, if I'm recalling correctly.

Reply:
Ah yes, I understand now. Cool, then this point sounds good to me.